books/models/country.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Country(models.Model):
+    name = models.CharField(max_length=250, null=False, unique=True)
```

Comment:
@xhahid43eb Please download a list of countries from any website, and create initial data population script to populate the country table. You may need to create some other fields in order to keep necessary data.

Reply:
Noted..